lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -76,6 +77,21 @@ data TransactionLayer t k = TransactionLayer
         -- This expects as a first argument a mean to compute or lookup private
         -- key corresponding to a particular address.
 
+    , mkDelegationCertTx
+        :: (Address -> Maybe (k 'AddressK XPrv, Passphrase "encryption"))
+        -> PoolId
+        -> (k 'AddressK XPrv, Passphrase "encryption")
+        -> [(TxIn, TxOut)]
+        -> [TxOut]
+        -> Either ErrMkStdTx (Tx, SealedTx)
+        -- ^ Construct a transaction containing a certificate for delegating to
+        -- a stake pool.
+        --
+        -- The certificate is a combination of the node public key ('PoolId'),
```

Comment:
node public key is the same as stake pool public key, right?

Reply:
Hm… good point, not sure. `public key` might be a too general term too (since there are so many kinds of keys)

Reply:
> node public key is the same as stake pool public key, right?

Yes it is.

Reply:
Made it "node VRF public key" at least.